packages/@angular/cli/tasks/serve.ts

```diff
@@ -182,7 +182,11 @@ export default Task.extend({
           return reject(err);
         }
         if (serveTaskOptions.open) {
-            opn(serverAddress);
+           if (serveTaskOptions.browser) {
+            opn(serverAddress, { app: serveTaskOptions.browser });
```

Comment:
From the link:
The app name is platform dependent. Don't hard code it in reusable modules. For example, Chrome is `google chrome` on macOS, `google-chrome` on Linux and `chrome` on Windows.